config/routes.rb

```diff
@@ -0,0 +1,51 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+
+  root to: 'homepages#index'
+
+  resources :homepages, only: [:index]
+
+  get '/trips', to: 'trips#index', as: 'trips'
+  get '/trips/new', to: 'trips#new', as: 'new_trip'
+  post '/trips', to: 'trips#create'
+
+  get '/trips/:id', to: 'trips#show', as: 'trip'
+  get '/trips/:id/edit', to: 'trips#edit', as: 'edit_trip'
+  patch '/trips/:id', to: 'trips#update'
+  delete '/trips/:id', to: 'trips#destroy'
```

Comment:
These could have also been `resources`:

```suggestion
  resources :trips
```